website/docs/d/epg_to_contract.html.markdown

```diff
@@ -15,22 +15,21 @@ Data source for ACI EPG to contract relationship.
 
 ```hcl
 data "aci_epg_to_contract" "example" {
-    application_epg_dn = aci_application_epg.demo.id
-    contract_name  = "example"
-    contract_type = "consumer"
+  application_epg_dn = aci_application_epg.example.id
+  contract_dn        = "example"
```

Comment:
if it is a dn, we need to put either a data source or resource interpolation in this example